bash_completion

```diff
@@ -1368,12 +1420,9 @@ _services()
     local sysvdirs
     _sysvdirs
 
-    local IFS=$' \t\n' reset=$(shopt -p nullglob)
-    shopt -s nullglob
-    COMPREPLY=(
-        $(printf '%s\n' ${sysvdirs[0]}/!($_backup_glob|functions|README)))
-    $reset
+    _comp_expand_glob COMPREPLY '${sysvdirs[0]}/!($_backup_glob|functions|README)'
 
+    local IFS=$' \t\n'
```

Comment:
I think this may have been added originally for `$reset` to work and thus might not be needed any longer -- at least this PR is removing it with another similar change in `_xinetd_services` above and elsewhere.

Reply:
Sorry for the delay. I think I kept it so that it doesn't affect the later word splitting `COMPREPLY+=($(...))`, and `IFS` removal in `_xinetd_services` is an oversight. 8b267ebf. I have also added `IFS=$'\n'` to protect `_service` for the splitting of `compgen` results.